app/src/main/java/com/willowtree/vocable/presets/PresetsFragment.kt

```diff
@@ -303,6 +304,19 @@ class PresetsFragment : BaseFragment() {
             }
             numPages = ceil(phrases.size / maxPhrases.toDouble()).toInt()
             notifyDataSetChanged()
+
+            if (phrasesAdapter.numPages == 1) {
+                setPagingButtonsEnabled(false)
+            } else {
+                setPagingButtonsEnabled(true)
+            }
+        }
+
+        private fun setPagingButtonsEnabled(enable: Boolean) {
+            binding.let {
```

Comment:
So the convention here is to do `binding?.let`, that way you can just call `it.` without the `?` each time.